object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/MustCallInvokedChecker.java

```diff
@@ -605,11 +772,13 @@ static String formatMissingMustCallMethods(List<String> mustCallVal) {
    */
   private static class BlockWithLocals {
     public final Block block;
-    public final Set<LocalVarWithTree> localSetInfo;
+    public final ImmutableSet<ImmutableSet<LocalVarWithTree>> localSetInfo;
 
-    public BlockWithLocals(Block b, Set<LocalVarWithTree> ls) {
+    public BlockWithLocals(Block b, Set<Set<LocalVarWithTree>> ls) {
```

Comment:
The type of `ls` should be `Set<ImmutableSet<LocalVarWithTree>>`.  Then we can get rid of lines 779 and 780